src/saleor_app/endpoints.py

```diff
@@ -59,25 +56,17 @@ async def install(
 
 async def handle_webhook(
```

Comment:
This endpoint now server no other cause than to be a description for the OAS3 spec